### PR TITLE
docs(infiniteGrid): remove fit method and itemSeletor options

### PR DESCRIPTION
**Pull request by sculove**
## Issue
#269 

## Details
- deprecated `fit` method
- remove itemSelector options

## Preferred reviewers
@kishu

### CODE REVIEW

**Comment by mixed** [comment]
@sculove 
Could you explain some more? 
Why InfiniteGrid have to deprecate `fit` and remove itemSelector options?

**Comment by sculove** [comment]
@mixed 
`fit` method doesn't give value to users because append and prepend events are added in egjs 1.1.0. we should have deprecated `fit` method since 1.1.0
it's so late.

`itemSelector` option hasn't been working since egjs 0.0.1
No one used this option.

**Comment by mixed** [comment]
@sculove 
I got it. LGTM.
But I think that this issue need discuss before merge. becacuse we have not deprecate process.

**Comment by jongmoon** [comment]
LGTM

File: src/infiniteGrid.js [hunk 1]
```diff
@@ -573,12 +572,6 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document], function($, ns, glob
 			return true;
 		},
 
-		/**
-		 * Remove white space which was removed by append action.
-		 * @ko append에 의해 생긴 빈공간을 제거한다.
-		 * @method eg.InfiniteGrid#fit
-		 * @return {Number} distance if empty space is removed, value is not zero. <ko>빈공간이 제거된 실제 길이를 px 단위로 반환</ko>
-		 */
 		fit: function() {
```
Comment (kishu) [inline src/infiniteGrid.js]: There is `_fit()` and `fit()` in this component.
Make it private function and change function name. Or you can combine `_fit()` and `fit()`.

Comment (sculove) [inline src/infiniteGrid.js]: 👍

**Pull request merged**